apps/src/sites/studio/pages/levels/levelGroup.js

```diff
@@ -53,10 +51,9 @@ window.initLevelGroup = function (levelCount, currentPage, lastAttempt) {
         app: subLevel.getAppName(),
         allowMultipleSends: true,
         level: subLevelId,
-        result: subLevelResult,
-        pass: subLevelResult,
-        testResult: testResult,
-        submitted: submitted,
+        result: true,
+        testResult: TestResults.UNVALIDATED_SUBLEVEL,
+        submitted: subLevelResult.submitted || false,
         onComplete: handleSublevelComplete
```

Comment:
Reasonably confident, as I've spent a fair amount of time investigating this.

`result` was previously an object with a bunch of subfields, but as far as I can tell these aren't checked anywhere (https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/controllers/activities_controller.rb#L19 is the interesting bit of code)
`pass` wasn't used at all afaict
`testResult` is what we're intentionally changing here
`submitted` is identical, just moved the code